libnymea-core/experiences/experiencemanager.h

```diff
@@ -45,9 +45,11 @@ class ExperienceManager : public QObject
 public:
     explicit ExperienceManager(ThingManager *thingManager, JsonRPCServer *jsonRpcServer, QObject *parent = nullptr);
 
-signals:
+    QList<ExperiencePlugin *> plugins() const;
+    void loadExperiencePlugin(const QString &file);
```

Comment:
is there a reason why you've made this public? AFAICT you'd need only the other one for testing...

Reply:
Load plugins...check if plugins have been loaded... I thought it makes only in combination sense, because the load method gives no feedback about the result.

Reply:
Yeah... it was meant as an internal function only... Not sure if it makes sense to allow loading plugins from a file from the outside... I can see why you'd want to load a plugin by object pointer to inject a mocked one but not really this, because you could also achieve the same by using `qPutEnv(NYMEA_EXPERIENCE_PLUGINS_PATH, ...)` and let the Experience manager do it's thing on his own...